install_config/install/deploy_router.adoc

```diff
@@ -75,9 +75,19 @@ $ oc create serviceaccount router -n default
 
 To add a privileged SCC to the *router* service account in the *default* namespace:
 
+====
 ----
 $ oadm policy add-scc-to-user privileged system:serviceaccount:default:router
 ----
+====
+
+Use of multiple name spaces requires cluster-reader premission.
```

Comment:
It's the use of labels that requires the permission.  So if the router definition sets `NAMESPACE_LABELS` then you need to give it the permission.